packages/flame/test/collisions/collision_test_helpers.dart

```diff
@@ -14,6 +18,35 @@ Future<void> testCollisionDetectionGame(
   return testWithGame(testName, HasCollidablesGame.new, testBody);
 }
 
+@isTest
+Future<void> testQuadTreeCollisionDetectionGame(
+  String testName,
+  Future Function(HasCollisionDetection) testBody,
+) {
+  return testWithGame(
+    testName,
+    () {
+      final game = HasQuadTreeCollidablesGame();
+      game.initializeCollisionDetection(
+        mapDimensions: const Rect.fromLTWH(0, 0, 1000, 1000),
+      );
+      return game;
+    },
+    testBody,
+  );
+}
+
+Future<void> runTestRegistry(
```

Comment:
```suggestion
Future<void> runCollisionTestRegistry(
```
It should have a name that is specific to collision detection since it only handles those cases at the moment.